R/grid_code_paths.R

```diff
@@ -538,7 +538,7 @@ super_safely <- function(fn) {
 
   # Construct a try()-error to be compatible with `log_problems()`
   handle_error <- function(e) {
-    e <- structure(e$message, class = "try-error", condition = e)
+    e <- structure(conditionMessage(e), class = "try-error", condition = e)
```

Comment:
This was unrelated to the finetune issue, but if the logic for the anologous fix there is sound, I figured we'd apply to the error place where this comes up!